hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/ContainerData.java

```diff
@@ -129,19 +129,20 @@
    * Creates a ContainerData Object, which holds metadata of the container.
    * @param type - ContainerType
    * @param containerId - ContainerId
-   * @param layOutVersion - Container layOutVersion
+   * @param layoutVersion - Container layoutVersion
    * @param size - Container maximum size in bytes
    * @param originPipelineId - Pipeline Id where this container is/was created
    * @param originNodeId - Node Id where this container is/was created
    */
   protected ContainerData(ContainerType type, long containerId,
-      ChunkLayOutVersion layOutVersion, long size, String originPipelineId,
-      String originNodeId) {
+                          ContainerLayoutVersion layoutVersion, long size,
+                          String originPipelineId,
+                          String originNodeId) {
     Preconditions.checkNotNull(type);
 
     this.containerType = type;
     this.containerID = containerId;
-    this.layOutVersion = layOutVersion.getVersion();
+    this.layOutVersion = layoutVersion.getVersion();
```

Comment:
Nit: Please also rename `this.layOutVersion` to `this.layoutVersion`.

Also in `DatanodeVersionFile`.

Reply:
The variable name `layOutVersion` itself is used to name the key inside the container data YAML file. Renaming this to `layoutVersion` will change the key persisted into the YAML to `"layoutVersion"` as well (potentially breaking backward compatibility).

Reply:
You are right, I forgot about that.